src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/include/ov_api_conformance_helpers.hpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "conformance.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+// TODO: fix namespaces
+using namespace ConformanceTests;
+
+namespace ov {
+namespace test {
+namespace conformance {
+
+inline const std::string getPluginLibNameByDevice(const std::string& deviceName) {
+    const std::map<std::string, std::string> devices{
+            { "AUTO", "AutoPlugin" },
+            { "HDDL", "HDDLPlugin" },
+            { "VPUX", "VPUXPlugin" },
+            { "AUTO", "ov_auto_plugin" },
+            { "CPU", "MKLDNNPlugin" },
+            { "GNA", "ov_intel_gna_plugin" },
+            { "GPU", "ov_intel_gpu_plugin" },
+            { "HETERO", "ov_hetero_plugin" },
+            { "MULTI", "ov_multi_plugin" },
+            { "MYRIAD", "myriadPlugin" },
+            { "TEMPLATE", "templatePlugin" },
```

Comment:
please, update this map. most of plugins are already renamed

Reply:
done